cpp/velox/jni/JniWrapper.cc

```diff
@@ -71,10 +72,10 @@ JNIEXPORT void JNICALL Java_io_glutenproject_vectorized_ExpressionEvaluatorJniWr
   JNI_METHOD_END()
 }
 
-JNIEXPORT void JNICALL Java_io_glutenproject_vectorized_ExpressionEvaluatorJniWrapper_nativeFinalizeNative(JNIEnv* env){
-    JNI_METHOD_START
-        // TODO Release resources allocated for single executor/driver
-        JNI_METHOD_END()}
+JNIEXPORT void JNICALL Java_io_glutenprovject_vectorized_ExpressionEvaluatorJniWrapper_nativeFinalizeNative(
+    JNIEnv* env){JNI_METHOD_START
+                     // TODO Release resources allocated for single executor/driver
```

Comment:
Please add an issue and paste issue link here

Reply:
This is not related changes. It seems format change.